packages/composite-checkout/src/components/checkout-order-summary.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import { useLineItems } from '../public-api';
+
+export default function CheckoutOrderSummary() {
+	const [ items ] = useLineItems();
+
+	return (
+		<React.Fragment>
```

Comment:
This component returns only a single element, `ProductList`, so it doesn't need to be wrapped in a `Fragment`. We only need to use Fragments when a component returns more than one element.

Reply:
oh yeah, I'm not sure why I left that in. Thanks!